nautobot/docs/additional-features/caching.md

```diff
@@ -1,9 +1,13 @@
 # Caching
```

Comment:
Maybe it's just me but Nautobot's use of caching is/was really confusing.  It seems at this point there is a better explanation of caching in required_settings then in caching.md itself.  Maybe explain why there are CACHES, RQ_QUEUES, and CACHEOPS what they are used for, why they are different, and which ones can use a shared DB and which ones need their own.  What does monkey patched mean to a non-software developer?  If Cacheops doesn't use the Django cache framework what does it use?  Just my $0.02

Reply:
I agree with that in this caching doc. I attempted to explain that in the settings doc but I think you're making a good point to go into even further detail in this doc.